benefits/templates/admin/includes/favicon.html

```diff
@@ -0,0 +1,3 @@
+{% load i18n static %}
+
+<link href="{% static "img/favicon.ico" %}" rel="icon" type="image/x-icon" />
```

Comment:
There is no `img/favicon.ico`. Maybe need a future ticket to make a new favicon? I don't think it should be the same favicon as https://benefits.calitp.org/.

cc @indexing

Reply:
Thanks @machikoyasuda! I'm looking into this to see where favicon gets set in the `index` admin page that we see when we log in as a superuser.

Reply:
Yes @machikoyasuda. Let's use the same favicon.

Reply:
Sounds good! I'll create a separate ticket to fix this favicon issue.